Bio/PDB/PDBList.py

```diff
@@ -455,15 +460,14 @@ def download_pdb_files(
                 overwrite=overwrite,
             )
 
-    def get_all_assemblies(self, file_format="mmCif"):
+    def get_all_assemblies(self, _file_format: str = "") -> List[Tuple[str, str]]:
```

Comment:
Probably best not to change the legacy argument name as that could break scripts.